rskj-core/src/main/java/co/rsk/net/NodeMessageHandler.java

```diff
@@ -112,6 +124,8 @@ public synchronized void processMessage(final Peer sender, @Nonnull final Messag
         else {
             loggerMessageProcess.debug("Message[{}] processed after [{}] seconds.", message.getMessageType(), timeInSeconds);
         }
+        
+        messageCounter.decrement(sender);
```

Comment:
what if `message.accept(mv)` method crashes? `messageCounter.decrement(sender)` wouldn't be called then..

Reply:
made decrement to be the first item on the method

Reply:
what if put `messageCounter.decrement(sender)` before `message.accept(mv)`?